tasks/internal_reconcile_report_phantom/test/unit_tests/test_internal_reconcile_report_phantom.py

```diff
@@ -0,0 +1,25 @@
+"""
+Name: test_internal_reconcile_report_phantom.py
+Description:  Unit tests for internal_reconcile_report_phantom.py.
+"""
+import datetime
+import json
+import random
+import unittest
+import uuid
+from unittest.mock import Mock, call, patch, MagicMock
+
+from fastjsonschema import JsonSchemaValueException
+
+import internal_reconcile_report_phantom
+from orca_shared.reconciliation import shared_reconciliation
+
+
+class TestInternalReconcileReportPhantom(
+    unittest.TestCase
+):  # pylint: disable-msg=too-many-instance-attributes
+    """
+    TestInternalReconcileReportPhantom.
+    """
+def test_dummy():
+    pass
```

Comment:
Since you have a dummy test, could we include this in run_tests instead of excluding it?

Reply:
yes updated bin/run_tests.sh

Reply:
Even though there literally isn't any code? Fun.